paperpi/library/Plugin.ipynb

```diff
@@ -77,6 +77,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "class PluginError(Exception):\n",
```

Comment:
I don't see where this is used. I like the idea of gathering all the Attribute/TypeErrors together under this exception class, but it's not used.

Reply:
Okay I will remove this class